package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "steamymedia",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Team-Gandalf/Media.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Team-Gandalf/Media/issues"
+  },
+  "homepage": "https://github.com/Team-Gandalf/Media#readme",
+  "dependencies": {
+    "axios": "^0.19.2",
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "nodemon": "^2.0.3",
```

Comment:
Is nodemon a dev dependency or a dependency? @sfshaider @joshskkim Do you think it matters?